superset/assets/javascripts/constants.js

```diff
@@ -0,0 +1 @@
+export const QUERY_TIMEOUT_THRESHOLD = 60000;
```

Comment:
seems like this should live in the sql lab constants file https://github.com/airbnb/superset/blob/dd4b2a5e2603e553c28c2967fb2d37b7d031510a/superset/assets/javascripts/SqlLab/constants.js